launch/client.py

```diff
@@ -3204,14 +3205,36 @@ def get_file_content(
         with ApiClient(self.configuration) as api_client:
             api_instance = DefaultApi(api_client)
             path_params = frozendict({"file_id": file_id})
-            response = api_instance.get_file_content_v1_files_file_id_content_get(  # type: ignore
+            response = api_instance.get_file_v1_files_file_id_get(  # type: ignore
```

Comment:
what's the reason we need to change this here? I'm sure there's a reason but want to make sure

Reply:
whoops, this actually really shouldn't be touched -- at least not for my pr. I think I must've been testing changes locally last week and playing with the files api, committed this by accident. I'm taking it out now.